draft/spec/index.html

```diff
@@ -485,6 +485,11 @@ <h2>Inventory</h2>
             <code>inventory.json</code>. The order of entries in both the [[JSON]] objects and arrays used in inventory
             files has no significance.
         </p>
+        <p>
+            The forward slash (/) path separator MUST be used in existing and logical file paths in the manifest,
```

Comment:
Should we add links to the `manifest`, `fixity`, and `state` sections of the spec?